ui/components/SubRouterTabs.tsx

```diff
@@ -96,4 +105,24 @@ export default styled(SubRouterTabs).attrs({ className: SubRouterTabs.name })`
   .active-tab {
     background: ${(props) => props.theme.colors.primary}19;
   }
+  .MuiTab-root {
+    line-height: 1;
+    letter-spacing: 1px;
+    height: 32px;
+    min-height: 32px;
+    width: 132px;
+    @media (min-width: 600px) {
+      min-width: 132px;
+    }
+  }
+  //trust me there's both tab and tabS
+  .MuiTabs-root {
+    min-height: 32px;
+  }
+  .MuiTabs-fixed {
+    height: 32px;
+  }
+  .PrivateTabIndicator-root-1 {
+    height 3px;
```

Comment:
Missing colon. And I don't see that this setting has an effect even when adding the colon. Maybe we should use

```
.MuiTabs-indicator {
  height: 3px;
}
```

instead?

Reply:
Wow...sorry not sure how the colon got messed up there. It seemed like PrivateTabIndicator was the correct class - I'm overriding the 2px value here. It looks different to me when I change the value but I could definitely be wrong.  
<img width="631" alt="image" src="https://user-images.githubusercontent.com/65822698/181353194-2c0fc82a-7e7d-454c-af4e-5702b9c2638e.png">

Reply:
2px screenshot:
<img width="148" alt="image" src="https://user-images.githubusercontent.com/65822698/181353555-f6debe25-1c9c-4717-9149-1190b107d58b.png">

3px screenshot:
<img width="145" alt="image" src="https://user-images.githubusercontent.com/65822698/181353646-6c48bc32-e85e-4e2a-9d47-5345c3fe37f2.png">

Reply:
How do you run the app? Via `npm start` or `tilt up`?

Because I'm using `tilt up` and don't see the `.PrivateTabIndicator-root-1` class in the inspector and the height of the blue border block for me is still 2px with the branch code.

But if I replace `.PrivateTabIndicator-root-1` with `.MuiTabs-indicator`, the height becomes 3px, as expected. 

Maybe `.PrivateTabIndicator-root-1` works only for `npm start`? Besides, here it says that it's a private class and should not be used

https://github.com/mui/material-ui/issues/21736#issuecomment-656402837

and I haven't found it in MUI docs:

With `.PrivateTabIndicator-root-1`:

<img width="1422" alt="Screenshot 2022-07-27 at 22 59 56" src="https://user-images.githubusercontent.com/8824708/181373327-26883922-1c1a-428c-b39f-63b112f8f7bc.png">

With `.MuiTabs-indicator`:

<img width="1496" alt="Screenshot 2022-07-27 at 23 08 43" src="https://user-images.githubusercontent.com/8824708/181373345-1c4a1ee8-f588-4453-b1fe-cdf8db7b495c.png">

Reply:
I also see another advice of using some props for this:

https://stackoverflow.com/questions/61394435/how-do-i-override-material-uis-tab-selection-color

Reply:
Changed the class to `.MuiTabs-indicator` - should be good to go!